ngraph/python/src/ngraph/__init__.py

```diff
@@ -20,6 +20,11 @@
 from ngraph.impl import FrontEnd
 from ngraph.impl import InputModel
 from ngraph.impl import Place
+from ngraph.impl import NotImplementedFailure
```

Comment:
these names are related to 'frontend' API, not to the whole ngraph. Is there a way to write something like "from ngraph.frontend import GeneralFailure"?

Reply:
done. Place,InputModel and others have also been moved.